src/raftstore/store/transport.rs

```diff
@@ -17,5 +17,5 @@ use raftstore::Result;
 
 // Transports message between different raft peers.
 pub trait Transport: Send + Sync {
-    fn send(&self, msg: RaftMessage) -> Result<()>;
+    fn send(&mut self, msg: RaftMessage) -> Result<()>;
```

Comment:
when I implement a Transport and call send, I need to change something in it's state, how to do that in rust?